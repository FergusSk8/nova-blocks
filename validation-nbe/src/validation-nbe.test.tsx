import React from 'react';
import { shallow, mount, ShallowWrapper } from 'enzyme';

import ValidationNbe from './index';
import FormItem from './components/FormItem';
import { Component } from 'react';

jest.mock('umi', () => {
  return {
    __esModule: true,
    useIntl: jest.fn(() => {
      return { formatMessage: jest.fn() };
    }),
  };
});

describe('Validation-Nbe', () => {
  let wrapper: ShallowWrapper;
  beforeEach(() => {
    wrapper = shallow(<ValidationNbe />);
  })

  test('should match with snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  // test('should have the correct className', () => {
  //   const myClass = wrapper.find('Form').at(0).hasClass('stepForm');
  //   expect(myClass).toBeTruthy();
  // })
  

  // test('should have Create as label', () => {
  //   const text = wrapper.find('Button').at(0).text();
  //   expect(text).toBe('Crear');
  // })
  
});
