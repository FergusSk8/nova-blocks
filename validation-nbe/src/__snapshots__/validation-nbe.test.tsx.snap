// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Validation-Nbe should match with snapshot 1`] = `undefined`;
